src/Elm/Parser/Tokens.elm

```diff
@@ -204,12 +204,12 @@ multiLineStringLiteral =
         helper s =
             if s.escaped then
                 escapedCharValue
-                    |> Core.map (\v -> Loop { s | escaped = False, parts = String.fromList [ v ] :: s.parts })
+                    |> Core.map (\v -> Loop { s | escaped = False, parts = String.fromChar v :: s.parts })
```

Comment:
This isn't necessary for the bug fix but it seemed strange to do `Char to List Char to String` instead of `Char to String`